### ISSUE TITLE
client group management -not loading

### ISSUE THREAD

**Issue by 91ajames** [pi-hole/docker-pi-hole#1114]
This is a: **Run Issue**  

## Related Issues
- [*] I have searched this repository/Pi-hole forums for existing issues and pull requests that look similar 

## How to reproduce the issue 

1. Environment data
  * Operating System: **Debian** 
  * Hardware: **Tinkerboard** 
  * Kernel Architecture: **4.4.132 ArmV7** 
  * Docker Install Info and version: 
    - Software source: **19.03.14** 
    - Supplimentary Software: **Portainer** 
  * Hardware architecture: **ARMv71** 

2. docker-compose.yml contents, docker run shell command, or paste a screenshot of any UI based configuration of containers here
3. any additional info to help reproduce
**Happened a few weeks ago, haven't touch much of the device for about 8months from testing out some projects.
Just simply won't load at all. Pi-Hole either just sits there loading... or gives these options under >Group Management>Clients**
**0**
**success**
**message (Error while quering FTL's database: not an error)**

**Have WatchTower to auto update Pi-Hole. Nothing seemed to change anything.**
**[Debug](https://tricorder.pi-hole.net/Heg6Un5H/)**
**DockerTag 2022.04.3 - Pi-Hole v5.10 - FTL v5.15 - WI v5.12**

## These common fixes didn't work for my issue

- [*] I have tried removing/destroying my container, and re-creating a new container

If the above debugging / fixes revealed any new information note it here.
Add any other debugging steps you've taken or theories on root cause that may help.

**Comment by rdwebdesign** [comment]
Are you able to access/view other admin pages?

**Comment by 91ajames** [comment]
Yes. Which is still odd. Tried on 2 other devices, even on the device itself or vnc.
I will try and back it up and just do a fresh install and then do a restore afterwards.

**Comment by rdwebdesign** [comment]
I suggest you first try a clean installation, in a new container, without volumes, using the fresh config files.

I this step works, you can try to restore your config.
